deploy/kubernetes-1.17/direct/pmem-csi.yaml

```diff
@@ -223,42 +303,27 @@ spec:
         pmem-csi.intel.com/deployment: direct-production
         pmem-csi.intel.com/webhook: ignore
     spec:
-      affinity:
-        nodeAffinity:
-          requiredDuringSchedulingIgnoredDuringExecution:
-            nodeSelectorTerms:
-            - matchExpressions:
-              - key: pmem-csi.intel.com/controller
-                operator: NotIn
-                values:
-                - "no"
-                - "false"
       containers:
       - command:
         - /usr/local/bin/pmem-csi-driver
         - -v=3
         - -logging-format=text
-        - -mode=controller
-        - -endpoint=unix:///csi/csi-controller.sock
-        - -registryEndpoint=tcp://0.0.0.0:10000
-        - -nodeid=$(KUBE_NODE_NAME)
+        - -mode=webhooks
+        - -schedulerListen=:8000
+        - -drivername=$(PMEM_CSI_DRIVER_NAME)
```

Comment:
We still need the external-provisioner in the controller as fallback for https://github.com/kubernetes-csi/external-provisioner/issues/544

Reply:
... or a custom controller which removes the selected-node annotation.

Reply:
... or just set `AllowedTopologies` in our storage classes for late binding (https://github.com/kubernetes-csi/external-provisioner/issues/544#issuecomment-748871274).